evergreen/config.yml

```diff
@@ -1670,12 +1499,12 @@ buildvariants:
     cmake_generator: Xcode
     max_jobs: $(sysctl -n hw.logicalcpu)
     xcode_developer_dir: /Applications/Xcode13.1.app/Contents/Developer
-    extra_flags: -DCMAKE_SYSTEM_NAME=Darwin -DCMAKE_OSX_ARCHITECTURES=arm64 -DREALM_ENABLE_GEOSPATIAL=OFF
```

Comment:
IIRC we do want one runner with geospatial turned off because the C++ SDK wants to make use of it to reduce binary size. ([slack](https://mongodb.slack.com/archives/CKS9713C4/p1710267554200229) context)

Reply:
Yup. Adding a builder to do that now.